conmon-rs/server/src/server.rs

```diff
@@ -239,6 +241,10 @@ impl Server {
             args.push(Self::SYSTEMD_CGROUP_ARG.into());
         }
 
+        if runtime_args.len() > 0 {
+            args.extend(runtime_args);
+        }
```

Comment:
I think we do not have to pre-check the length:
```suggestion
        args.extend(runtime_args);
```